mutt_thread.c

```diff
@@ -679,29 +680,28 @@ static int compare_threads(const void *a, const void *b)
  * @param tctx Threading context
  * @param init If true, rebuild the thread
  */
-void mutt_sort_subthreads(struct ThreadsContext *tctx, bool init)
+static void mutt_sort_subthreads(struct ThreadsContext *tctx, bool init)
 {
   struct MuttThread *thread = tctx->tree;
   if (!thread)
     return;
 
   struct MuttThread **array = NULL, *sort_key = NULL, *top = NULL, *tmp = NULL;
   struct Email *oldsort_key = NULL;
-  int i, array_size, sort_top = 0;
+  int i, array_size;
+  bool sort_top = false;
 
   /* we put things into the array backwards to save some cycles,
    * but we want to have to move less stuff around if we're
    * resorting, so we sort backwards and then put them back
    * in reverse order so they're forwards */
-  short c_sort = cs_subset_sort(NeoMutt->sub, "sort");
+  short c_sort = cs_subset_sort(NeoMutt->sub, "sort_aux");
```

Comment:
Please can you change `c_sort` to `c_sort_aux`

Reply:
updated